test/parallel/test-fs-read-empty-buffer.js

```diff
@@ -26,3 +27,14 @@ assert.throws(
     'Received Uint8Array []'
   }
 );
+
+fsPromises.open(filepath, 'r').then((filehandle) => {
```

Comment:
Also, now that you are doing it, please cover the fs promises read with a file descriptor.